src/util/document_apis.jsx

```diff
@@ -0,0 +1,35 @@
+import AllDocs from "@/util/test_documents";
+
+/*
+    Gets all information on all docs
+*/
+function getAllDocs() {
+    return AllDocs
+}
+
+/*
+    Gets ids, name, and dateLastModified (and other metadata) info about all docs
+*/
+function getAllDocIdsAndMetadata() {
+    return AllDocs.map(doc => {
+        var docMetadata = doc
+        delete docMetadata['body']
+        return docMetadata
+    })
+    //Note: When backend is setup, this should only call the api and return doc metadata
+}
+
+/*
+    Get metadata of doc given it's id
```

Comment:
hehe nit but `its` not `'it's` no apostrophe in the possessive ;) (sorry)

Reply:
I refuse to accept this socially, but code wise i'll make that change